amcl/test/basic_localization_stage.xml

```diff
@@ -47,5 +47,5 @@ setting pose: 30.329 34.644 3.142
       <param name="initial_pose_a" value="-1.003"/>
     </node>
     <test time-limit="180" test-name="basic_localization_stage" pkg="amcl"
-          type="basic_localization.py" args="0 30.329 34.644 3.142 0.75 0.75 90.0"/>
+          type="basic_localization.py" args="0 30.329 34.644 3.142 0.75 0.75 120.0"/>
```

Comment:
I'm curious. Why this parameter change is conducted.

Reply:
Since in this change we presume that there will be connectivity issues so I increased the target test time.

If you don't think this is necessary then please let me revert.

Reply:
Ofcourse, thank you for your comment.